portfolio/src/main/webapp/script.js

```diff
@@ -126,9 +126,29 @@ function loadComments() {
   });
 }
 
-// Creates a list element with the given text
-function createListElement(txt) {
+// Creates a list element with the given comment text and metadata (name, timestamp etc.)
+function createListElement(comment) {
   const listElem = document.createElement("li");
-  listElem.innerText = txt;
+  const metadata = formatCommentMetadata(comment);
+  const quote = formatCommentText(comment);
+  listElem.appendChild(metadata);
+  listElem.appendChild(quote);
+  listElem.className = "comment";
   return listElem;
+}
+
+// Formats comment name and timestamp into an HTML p element
+function formatCommentMetadata(comment) {
+  const metadata = comment['name'] + ' at ' + comment['timestamp'] + ' said:';
```

Comment:
You could use template literals (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) instead of string concat. May make it slightly more readable.

Also be consistent with "single quote" vs "double quote" :) Other parts of the code use double quote, so it is better to also do that here.